src/app/screens/Settings.tsx

```diff
@@ -182,6 +206,26 @@ function Settings() {
             </div>
           )}
         </Setting>
+
+        <Setting
+          title="Change Password"
```

Comment:
This is missing the i18 string. The label button below as well... sorry

Reply:
Oops, sorry! Done!